cmd/list.go

```
@@ -53,7 +54,7 @@ func init() {
 	ListCmd.Flags().VarP(&listOutputFormat, "format", "F", "Set the output format (list|json|yaml)")
 	ListCmd.Flags().BoolVar(&showCache, "cache-info", false, "Show cache information and exit")
 
-	ListCmd.RegisterFlagCompletionFunc("format", completionFormatData)
+	_ = ListCmd.RegisterFlagCompletionFunc("format", completionFormatData)
```

See comment above regarding ignoring errors.